src/App.tsx

```diff
@@ -15,35 +20,91 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+function getGoods(
+  goods: string[],
+  sortField: SortType | '',
+  reverse: boolean,
+): string[] {
+  const newGoods = [...goods];
+
+  if (sortField !== '') {
+    newGoods.sort((a, b) => {
+      switch (sortField) {
+        case SortType.Alphabetically:
+          return a.localeCompare(b);
+
+        case SortType.Length:
+          return Number(a.length) - Number(b.length);
+
+        default:
+          return 0;
+      }
+    });
+  }
+
+  return reverse ? newGoods.reverse() : newGoods;
+}
+
 export const App: React.FC = () => {
+  const [reverse, setReverse] = useState(false);
+  const [sortValue, setSortValue] = useState(-1);
```

Comment:
The `sortValue` state is initialized to `-1`, which is inconsistent with the expected type `SortType | ''`. Consider initializing it to `''` or a valid `SortType` value to match the expected type.